draft-ietf-quic-transport.md

```diff
@@ -1477,35 +1481,240 @@ connection if the integrity check fails with a PROTOCOL_VIOLATION error code.
 
 ## Connection Migration {#migration}
 
-QUIC connections are identified by their 64-bit Connection ID.  QUIC's
-consistent connection ID allows connections to survive changes to the client's
-IP and/or port, such as those caused by client or server migrating to a new
-network.  Connection migration allows a client to retain any shared state with a
-connection when they move networks.  This includes state that can be hard to
-recover such as outstanding requests, which might otherwise be lost with no easy
-way to retry them.
+QUIC allows connections to survive changes to endpoint addresses (that is, IP
+address and/or port), such as those caused by a client migrating to a new
+network.  This section describes the protocol for migrating a connection to a
+new client address.
+
+Migrating a connection to a new server address is left for future work. If a
+client receives packets in the connection from a new server address, the client
```

Comment:
nit: I don't think "in the connection" is necessary, since you begin the sentence with "Migrating a connection..."

Reply:
I'm in favor of Ian's change, because if packets don't map to a connection they'll be discarded anyway.  The extra words just sound awkward.  However, it's not technically incorrect, so this is purely wordsmithing.

Reply:
Ok, removed.